address_book/contacts/views.py

```diff
@@ -0,0 +1,38 @@
+from django.shortcuts import render, redirect
```

Comment:
Please use proper conventions for imports as per pep8.

Reply:
updated